### PR TITLE
Added few new sections, modified few sections and added git action for release creation #minor

### PR DESCRIPTION
**Pull request by ShubhangiGaherwar**
…lease creation #minor

### CODE REVIEW

File: .github/workflows/release_creation.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+        with:
+          tag_name: ${{ steps.bump_version_and_push_tag.outputs.new_tag }}   #steps.bump_version_and_push_tag.outputs.new_tag github.ref
+          release_name: Release ${{ steps.bump_version_and_push_tag.outputs.new_tag }}
+          body: |
+            Changes in this Release
+            - First Change
+            - Second Change
```
Comment (faizsiddiqui) [inline .github/workflows/release_creation.yml:40]: Change body to -

Release ${{ steps.bump_version_and_push_tag.outputs.new_tag }}

File: controls/4.3.rb [hunk 2]
```diff
@@ -22,6 +22,7 @@
   end
 
   describe command('ps aux | grep mysql') do
+    puts command('ps aux | grep mysql').stdout
```
Comment (faizsiddiqui) [inline controls/4.3.rb:25]: Please remove unnecessary puts commands.

**Review by faizsiddiqui — CHANGES_REQUESTED** [review]

File: .github/workflows/release_creation.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+name : release creation action
+on: push
```
Comment (faizsiddiqui) [inline .github/workflows/release_creation.yml:2]: Change this to -

```
on:
  push:
    branches:
      - main

**Review by faizsiddiqui — CHANGES_REQUESTED** [review]

File: .github/workflows/release_creation.yml (hunk 1 above)
Comment (ShubhangiGaherwar) [reply .github/workflows/release_creation.yml]: Changes done!

File: controls/4.3.rb (hunk 2 above)
Comment (ShubhangiGaherwar) [reply controls/4.3.rb]: removed it

File: .github/workflows/release_creation.yml (hunk 3 above)
Comment (ShubhangiGaherwar) [reply .github/workflows/release_creation.yml]: Updated it..
Thank you!

**Review by faizsiddiqui — APPROVED** [review]

**Pull request merged**